src/main/js/components/TableView.jsx

```diff
@@ -76,12 +85,100 @@ class TableView extends React.Component {
       .map(f.prop(["tableViews"]))
       .map(JSON.parse)
       .getOrElse({});
-    localStorage["tableViews"] = JSON.stringify(f.set([currentTableId,name], view, savedViews))
+    localStorage["tableViews"] = JSON.stringify(f.set([currentTableId, name], view, savedViews));
+  };
+
+  cellJumpError = msg => {
+    ActionCreator.showToast(
+      <div id="cell-jump-toast">{msg}</div>,
+      7000
+    )
+  };
+
+  checkGotoCellRequest = (loaded) => {
+    if (!this.pendingCellGoto) {
+      return;
+    }
+
+    const column = this.pendingCellGoto.column;
+    const columns = this.getCurrentTable().columns.models;
+    if (!this.checkIfColExists(columns, column)) {
+      return;
+    }
+    const {page} = this.pendingCellGoto;
+    if (loaded >= page || this.tableFullyLoaded) {
+      this.gotoCell(this.pendingCellGoto, loaded);
+    }
+  };
+
+  checkIfColExists = (columns, colId) => {
+    if (f.findIndex(f.matchesProperty("id", colId), columns) < 0) {
+      this.cellJumpError(i18n.t("table:jump.no_such_column", {col: colId}));
+      this.pendingCellGoto = null;
+      false;
+    } else {
+      return true;
+    }
+  };
+
+  estimateCellPage = row => 1 + Math.ceil((row - INITIAL_PAGE_SIZE) / PAGE_SIZE);
+
+  gotoCell = ({row, column, page, filter, ignore = false}, nPagesLoaded = 0) => {
+    if (! this.checkIfColExists(this.getCurrentTable().columns.models, column)) {
+      return;
+    }
+    const cellId = `cell-${this.state.currentTableId}-${column}-${row}`;
+
+    // Helper closure
+    const focusCell = cell => {
+      this.setColumnsVisibility({
+        val: true,
+        coll: [column]
+      });
+      if (filter) {
+        this.changeFilter({
+          filterMode: FilterModes.ID_ONLY,
+          filterValue: row,
+          filterColumnId: "noop",
+          sortColumnId: 0
+        });
+      }
+      const rows = this.getCurrentTable().rows.models;
+      const rowIndex = f.findIndex(f.matchesProperty('id', row), rows);
+      const columns = this.getCurrentTable().columns.models;
+      const visibleColumns = columns.filter(x => x.visible);
+      const colIndex = f.findIndex(f.matchesProperty("id", column), visibleColumns);
+      const scrollContainer = f.first(document.getElementsByClassName("data-wrapper"));
+      const xOffs = ID_CELL_W + (colIndex) * CELL_W - (window.innerWidth - CELL_W) / 2;
+      const yOffs = (filter)
+        ? 0
+        : CELL_H * rowIndex - (scrollContainer.getBoundingClientRect().height - CELL_H) / 2;
+      scrollContainer.scrollLeft = xOffs;
+      scrollContainer.scrollTop = yOffs;
+
+      ActionCreator.toggleCellSelection(cell, ignore, this.props.langtag);
+      return cell;
+    };
+
+    if (nPagesLoaded >= page || this.tableFullyLoaded) {
+      either(this.getCurrentTable().rows)
+        .map(rows => rows.get(row).cells)
+        .map(cells => cells.get(cellId))
+        .map(focusCell)
+        .orElse(() => this.cellJumpError(i18n.t("table:jump.no_such_row", {row: row})));
```

Comment:
Works fine for newly loaded tables. But the URL doesn't change if someone deletes a row. To delete a row you need to focus a cell of this row. Same if someone sets a filter or changes the visible columns. Cell focus is lost then but URL doesn't change.